src/widget_macro.md

```diff
@@ -52,6 +52,29 @@ Sometimes we want to use a constructor function to initialize our widgets. For t
 {{#include ../examples/simple.rs:widget_assign_fn }}
 ```
 
+### Returned Widget Syntax
+
+Some methods also return a newly created widget which we want to adjust to our needs.
+E.g. `Stack`'s [`add_child`](https://gtk-rs.org/gtk4-rs/stable/latest/docs/gtk4/struct.Stack.html#method.add_child) method returns a `StackPage` where we can set if the page needs [special user attention](https://gtk-rs.org/gtk4-rs/stable/latest/docs/gtk4/struct.StackPage.html#method.needs_attention):
+
+![App screenshot dark](img/stack_switcher.png)
+
+To enable this, there is an extension of the described syntax, where we can apply additional changes to the returned widget:
+
+```rust,no_run,noplayground
+add_child = &gtk::Label {
+    set_label: "Stopwatch",
```

Comment:
Setting the label to "Stopwatch" doesn't make a lot of sense. Maybe something like "42 seconds have passed". After all, that's the content of your page.